src/sas/cli.py

```diff
@@ -0,0 +1,171 @@
+#
+#Also see sasview\src\sas\qtgui\Perspectives\Fitting\media\cli.rst
+#
+"""
+**SasView command line interface**
+
+This functionality is under development. Interactive sessions do not yet
+work in the Windows console.
+
+**Usage:**
+
+sasview [flags]
```

Comment:
What does this look like once rendered by sphinx? Would expect to have seen some more formatting on these to make a good looking output following the normal conventions for commands and options,  but perhaps it looks ok as it is.

Reply:
No idea. I can't get the docs to build on my machine.